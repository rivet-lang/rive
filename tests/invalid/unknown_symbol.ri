enum MyEnum {
    Musk
}

func main() {
    unknown_fn();
    x := UnknownStruct();
    z := unknown_var;
    _ = MyEnum.Mark;
}
