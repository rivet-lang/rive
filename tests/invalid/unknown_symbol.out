tests/invalid/unknown_symbol.ri:6:5: error: cannot find `unknown_fn` in this scope
  6 |     unknown_fn();
    |     ^
tests/invalid/unknown_symbol.ri:7:13: error: cannot find `UnknownStruct` in this scope
  7 |     let x = UnknownStruct();
    |             ^
tests/invalid/unknown_symbol.ri:8:13: error: cannot find `unknown_var` in this scope
  8 |     let z = unknown_var;
    |             ^
tests/invalid/unknown_symbol.ri:9:16: error: enum `MyEnum` has no value `Mark`
  9 |     _ = MyEnum.Mark;
    |                ^
rivetc: error: could not compile module `unknown_symbol`, aborting due to 4 previous errors